### PR TITLE
@cldougl Created doc for 2d histo contour

### PR DESCRIPTION
**Pull request by ellecj**

### CODE REVIEW

File: _posts/plotly2/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+permalink: /make-a-2d-histogram-contour-plot/
+imageurl: ../static/images/create-contour-plot.md/2d contour thumb.png
+state: active
+tags: statistical
+order: 2
+meta_description: A tutorial on how to make a 2D Histogram Contour Plot in Plotly 2.0.
+popularity: featured
+carouselimageurl:
+actioncall: How to Make a 2D Histogram Contour Plot in Plotly 2.0
+actioncall-url: https://plot.ly/alpha/workspace/?fid=plotly2_demo:202
```
Comment (cldougl) [inline _posts/plotly2/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: Cool graph!!! Want to add latex to the x-axis label? If you type $sin^2$ the 2 will become a proper squared superscript.

File: _posts/plotly2/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+otherlang: Know how to program? See how to create this in [Python](https://plot.ly/python/density-plots/#2d-histogram-contour-plot-with-histogram-subplots).
+
+live-graph: <iframe width="900" height="800" frameborder="0" scrolling="no" src="https://plot.ly/~plotly2_demo/202.embed"></iframe>
+live-graph-image:
+
+steps:
+ - title: Try an Example
+   sub-steps:
+    - copy: "This 2 dimensional histogram compares distributions across two variables in a contour plot fashion."
```
Comment (cldougl) [inline _posts/plotly2/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: Can you add 1 sentence here that the distribution is depicted by a colorscale.

**Comment by cldougl** [comment]
good to 💃 after extra sentence about distribution = colorscale

File: _posts/plotly2/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md (hunk 1 above)
Comment (ellecj) [inline _posts/plotly2/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: Ooh!!! I forgot about using latex. Good thinking.

**Pull request merged**